### ISSUE TITLE
tor-browser-bundle-bin: Some website tabs crash reproducibly

### ISSUE THREAD

**Issue by schmittlauch** [NixOS/nixpkgs#95665]
**Describe the bug**

When trying to open some web pages in tor-browser-bundle-bin, the tab reproducibly crashes after load.
This issue does not appear in the Windows build or when running the Linux64 build under Ubuntu, so this bug appears to be related to the Nix packaging.
It also isn't related to a broken browser profile, as deleting `~/.local/share/tor-browser` does *not* resolve the issue.

Link to upstream bug: https://gitlab.torproject.org/tpo/applications/tor-browser/-/issues/33186

**To Reproduce**
1. `nix-shell -p tor-browser-bundle-bin --run tor-browser`
2. Try opening one of the websites https://facebookcorewwwi.onion/ http://planet.blogsport.de/
3. See the tab crash after loading has finished

**Screenshots**
![Screenshot_20200817_104618](https://user-images.githubusercontent.com/1479555/90376531-e6226d00-e076-11ea-985a-9fd20be9b8df.png)

**Additional context**
tor-browser-bundle-bin-9.5.3 from nixpkgs 20.03.2790.b4dac6714d7

**Notify maintainers**

@offlinehacker @matejc @doublec @thoughtpolice @joachifm @hax404 @scaredmushroom ( @andir for firefox knowledge) 

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: tor-browser-bundle-bin
# a list of nixos modules affected by the problem
module:
```

**Comment by marius851000** [comment]
this issue doesn't happen for tor-browser-bundle-bin-9.5.3 NixOS, 20.09.20200805.6a0fd33 , but I was able to reproduce it with your version.

**Comment by marius851000** [comment]
Found this was solved with the commit 45e2ff349fab4f61cb1cc6f4a331b47dec9b0bec
It explicitly say:
```

    * tor-browser: disable hardening by default
    
    this seems to cause crashes with certain tabs.
    relevant issue: https://github.com/NixOS/nixpkgs/issues/86356
    
    * Update pkgs/applications/networking/browsers/tor-browser-bundle-bin/default.nix
    
    Co-authored-by: Jörg Thalheim <Mic92@users.noreply.github.com>
```

**Comment by marius851000** [comment]
backported here: https://github.com/NixOS/nixpkgs/pull/95688

**Comment by schmittlauch** [comment]
fixed by #95688.
